core/src/main/scala/pureconfig/BasicConverters.scala

```diff
@@ -0,0 +1,106 @@
+package pureconfig
+
+import java.net.{ URI, URL }
+import java.nio.file.{ Path, Paths }
+import java.time._
+import java.util.UUID
+
+import scala.concurrent.duration.{ Duration, FiniteDuration }
+
+import com.typesafe.config._
+import pureconfig.error._
+
+/**
+ * Trait containing `ConfigConvert` instances for primitive types and simple classes in Java and Scala standard
+ * libraries.
+ */
+trait BasicConverters extends ConvertHelpers {
+
+  // primitive type converters
```

Comment:
(Please pardon the following grumpiness. I'm just expressing a preference. If my suggestion seems crazy, don't worry about it.)


I kind of loathe comments used like this to demarcate blocks of similar things. I'd rather see types used to group similar things.

My preference would be for something more like:

```
trait PrimativeConverters {
...
}

trait UriAndPathConverters {
  ...
}

trait JavaTimeConverters {
...
}

// etc

trait BasicConverters 
  extends PrimativeConverters 
  with UriAndPathConverters
  with JavaTimeConverters
  with ... {
...
}

object BasicConverters extends BasicConverters {
}
```